commit 9eeaee1fe60bedf719afc1c0c2ea94f27f852666
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Mon Oct 29 15:55:42 2018 +0000

    staging: bcm2835-camera: Fix alignment should match open parenthesis
    
    Fix up checkpatch "Alignment should match open parenthesis" errors
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c b/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
index f1fb11969a61..44a4c36904aa 100644
--- a/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
+++ b/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
@@ -1934,7 +1934,7 @@ static int bcm2835_mmal_probe(struct platform_device *pdev)
 		ret = bm2835_mmal_init_controls(dev, &dev->ctrl_handler);
 		if (ret < 0) {
 			v4l2_err(&dev->v4l2_dev, "%s: could not init controls: %d\n",
-				__func__, ret);
+				 __func__, ret);
 			goto unreg_dev;
 		}
 		dev->v4l2_dev.ctrl_handler = &dev->ctrl_handler;
@@ -1944,7 +1944,7 @@ static int bcm2835_mmal_probe(struct platform_device *pdev)
 		ret = mmal_init(dev);
 		if (ret < 0) {
 			v4l2_err(&dev->v4l2_dev, "%s: mmal init failed: %d\n",
-				__func__, ret);
+				 __func__, ret);
 			goto unreg_dev;
 		}
 		/* initialize queue */
@@ -1966,7 +1966,7 @@ static int bcm2835_mmal_probe(struct platform_device *pdev)
 		ret = bm2835_mmal_init_device(dev, &dev->vdev);
 		if (ret < 0) {
 			v4l2_err(&dev->v4l2_dev, "%s: could not init device: %d\n",
-				__func__, ret);
+				 __func__, ret);
 			goto unreg_dev;
 		}
 
@@ -1976,7 +1976,7 @@ static int bcm2835_mmal_probe(struct platform_device *pdev)
 		ret = mmal_setup_components(dev, &default_v4l2_format);
 		if (ret < 0) {
 			v4l2_err(&dev->v4l2_dev, "%s: could not setup components: %d\n",
-				__func__, ret);
+				 __func__, ret);
 			goto unreg_dev;
 		}
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1364_9eeaee1fe60bedf719afc1c0c2ea94f27f852666.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1364_9eeaee1fe60bedf719afc1c0c2ea94f27f852666.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
